eventing/samples/github-source/README.md

```diff
@@ -100,6 +100,10 @@ Source for a specific namespace. Be sure to replace the
 `ownerAndRepository` value with a valid GitHub public repository owned
 by your GitHub user.
 
+If using GitHub enterprise you will need to add an additional githubAPIURL
+field to the spec specifying your GitHub enterprise API endpoint, see 
+[here](https://github.com/knative/docs/tree/master/eventing#githubsource)
```

Comment:
Please use a relative link instead, since that page is within the docs repo.

Reply:
Good spot! - changed in latest commit.